lib/rabbitmq/cli/core/helpers.ex

```diff
@@ -187,6 +183,16 @@ defmodule RabbitMQ.CLI.Core.Helpers do
     :net_adm.ping(node) == :pong
   end
 
+  def rabbit_app_running?(%{node: node, timeout: timeout}) do
+    case :rabbit_misc.rpc_call(node, :rabbit, :is_running, [], timeout) do
+      true -> :ok
+      false ->
+        {:validation_failure, {:rabbit_app_not_running, node}}
+      error ->
```

Comment:
This breaks one test: if you run a command with `-t 0`, which makes it immediately time out, the error returned confuses error formatting and it exits with the "bad usage" code (64) instead of a "temporary failure" (75) which is probably more suitable. A test in `rabbitmqctl_test.exs` catches this.

We should not wrap RPC errors as `:validation_failure` or they'd mislead the user (by saying that the rabbit app isn't running, for example).